kazoo/tests/test_connection.py

```diff
@@ -279,7 +278,15 @@ def test_read_only(self):
         from kazoo.exceptions import NotReadOnlyCallError
         from kazoo.protocol.states import KeeperState
 
-        client = self.client
+        if CI:
+            # force some wait to make sure the data produced during the
+            # `setUp()` step are replicaed to all zk members
```

Comment:
the casing of `setUp()` looks like Java, should it be changed to `setup()` ?? Or is that intentional... ?

Reply:
It is intentional as I am referring to the `setUp()` method from the `TestCase` class (https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUp)